README.md

```diff
@@ -299,6 +299,28 @@ jobs:
             running-workflow-name: wait-for-check-regexp
             check-regexp: .?-task
   ```
+  ### Ignore-checks
+  To selectively filter checks and ignore specific ones, you can specify the ignore-checks option with a list of comma-separated check names to be ignored.
+  Example of use:
+  ```yaml
+  name: Wait using check-regexp
+  on:
+    push:
+
+  jobs:
+    wait-for-check-regexp:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Wait on tests
+          uses: ./
+          with:
+            ref: ${{ github.sha }}
+            repo-token: ${{ secrets.GITHUB_TOKEN }}
+            running-workflow-name: wait-for-check-regexp
+            ignore-checks: "enforcing labels"
```

Comment:
could you add an example where an array of checks is used? and probably adding a test workflow for this feature

Reply:
Done ! added

Reply:
could you add a simple test workflow? It not only works as test but as documentation/example as well.